go/src/socialapi/workers/common/mux/mux.go

```diff
@@ -37,6 +38,9 @@ type Mux struct {
 	config *Config
 	log    logging.Logger
 	redis  *redis.RedisSession
+
+	closing bool
+	closeMu sync.RWMutex
```

Comment:
`RWMutex` is not needed here, we're not expecting multiple readers and we shouldn't care - `sync.Mutex` is enough